Create 3_manual_voting.md (#674)
dfinity/portal: roadmap/4_governance_nns/future/3_manual_voting.md

---

> Title: Proposal to Enable Manual Voting Throughout the Entire Voting Period of Governance Proposals
> Links
> Forum: https://forum.dfinity.org/t/proposal-to-enable-manual-voting-throughout-the-entire-voting-period-of-governance-proposals/9815
> Proposal: https://dashboard.internetcomputer.org/proposal/38985

Reviewer: ```suggestion
title: Proposal to Enable Manual Voting Throughout the Entire Voting Period of Governance Proposals
links:
  Forum Link: https://forum.dfinity.org/t/proposal-to-enable-manual-voting-throughout-the-entire-voting-period-of-governance-proposals/9815
  Proposal: https://dashboard.internetcomputer.org/proposal/38985
```

is_community: true
---

> It is suggested to enable manual voting throughout the entire voting period of governance proposals.

Reviewer: ```suggestion
This feature enables manual voting throughout the entire voting period of governance proposals.
```
